check_format.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+const childProcess = require('child_process');
+const fs = require('fs');
+const glob = require('glob');
+
+const CLANG_FORMAT = 'node_modules/.bin/clang-format';
```

Comment:
I looked into it and it exposes a function that involves callbacks rather than running sync (while amusingly also using a .existsSync within it, so not really async) which makes it hard to use.  I'll leave that for future work.